src/main.rs

```diff
@@ -374,8 +374,9 @@ async fn add(plugin: Plugin<State>, args: AddArgs) -> Result<serde_json::Value,
         .map_err(|e| anyhow!("Error calling listdatastore: {:?}", e))?;
     let name = &dw.get_name()?;
     let message = format!("Wallet with deterministic name {} successfully added", name);
+    let db_path = dw_clone.get_db_file_path(db_dir).unwrap();
     log::info!("{}", message);
-    Ok(json!({"name": name, "message": message}))
+    Ok(json!({"name": name, "message": message, "db_path": db_path}))
```

Comment:
Why are we outputting this to the user?

Reply:
Currently, the returned `db_path` from the `add` RPC is used in the `remove` RPC test in order to be able to check for the existence of the smaug db file: `assert os.path.isfile(db_path)` and `assert not os.path.isfile(db_path)`, resp.
But yes, I am aware it is a bad idea to expose the db path in this way just to be able to accomplish testing. What would be a better way to accomplish this?

Reply:
Just calculate the path the same way we calculate it in smaug.

Should just be $LIGHTNING_DIR/.smaug/<wallet-name>.db

Reply:
Fixed.